all-spark-notebook/Dockerfile

```diff
@@ -3,8 +3,6 @@ FROM jupyter/minimal-notebook
 
 MAINTAINER Jupyter Project <jupyter@googlegroups.com>
 
-USER root
```

Comment:
Is this assumed to be root at this point? We are running `apt-get` right after, so we might as well be explicit.